client/e2e/testData/dataConstants/EventInfoConstants.ts

```diff
@@ -5,6 +5,7 @@ export const EVENT_INFO_1: EventInfo = {
   eventDescription: '123',
   eventURL: 'http://',
   eventFullName: 'S Playbill S Description 123',
+  suffix:"Playbill"
```

Comment:
Remove all suffixes. It does not logically fit this data object.  You can create a Constant called SUFFIX, in your test file and convert any string to add this suffix in.

Reply:
or a local variable within the function in which you use it. You only use it in one function. It does not warrant modifying the data constants like this.